### PR TITLE
migrate dynamo sandbox

### PR DESCRIPTION
**Pull request by pinzart90**
Migrate DynamoSandbox to dotnet6

### CODE REVIEW

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -108,19 +108,30 @@
...
+    <Reference Include="System.Core" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Web" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xaml" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Xml" />
+    <Reference Include="WindowsBase" />
```
Comment (aparajit-pratap) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:128]: These are all implicitly referenced in net6.0 or some such thing?

Comment (pinzart90) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:128]: They come automatically when targeting net6-windows + UseWPF
Projects will get a reference to the Microsoft.WindowsDesktop.App package.

You can see all the assembly references under ref/net6 subfolder here:
https://nuget.info/packages/Microsoft.WindowsDesktop.App.Ref/6.0.14

File: src/Dynamo.All.Net6.sln [hunk 2]
```diff
@@ -124,6 +124,7 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PackageManagerTests", "..\test\Libraries\PackageManagerTests\PackageManagerTests.csproj", "{2A70F3B5-7AEA-42FE-8DD5-8427C28D0025}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamoMSOfficeTests", "..\test\Libraries\DynamoMSOfficeTests\DynamoMSOfficeTests.csproj", "{269039F1-126A-46CF-8BEA-257587EDCA6B}"
+EndProject
```
Comment (mjkkirschner) [inline src/Dynamo.All.Net6.sln:127]: manually added?

File: src/Dynamo.All.Net6.sln [hunk 3]
```diff
@@ -864,6 +867,22 @@ Global
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|NET60_Windows.Build.0 = Release|NET60_Windows
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.ActiveCfg = Release|x64
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.Build.0 = Release|x64
+		{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (mjkkirschner) [inline src/Dynamo.All.Net6.sln:870]: I've been meaning to remove these other platforms (x64,anycpu) I guess it doesn't get harder the longer we wait with the configuration manager ui.

File: src/DynamoSandbox/DynamoSandbox.csproj [hunk 4]
```diff
@@ -18,11 +21,12 @@
   <PropertyGroup>
     <ApplicationIcon>logo_square_32x32.ico</ApplicationIcon>
   </PropertyGroup>
-
   <ItemGroup>
     <Reference Include="Microsoft.Practices.Prism">
       <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```
Comment (mjkkirschner) [inline src/DynamoSandbox/DynamoSandbox.csproj:26]: so does this actually still function without being updated?

Comment (pinzart90) [reply src/DynamoSandbox/DynamoSandbox.csproj:26]: it seems to. Unless we get a version of it specifically built for net6. we won't have a lot of guarantees that all APIs are compatible.

File: src/Dynamo.All.Net6.sln (hunk 3 above)
Comment (pinzart90) [reply src/Dynamo.All.Net6.sln:870]: I;ve been meaning to do the same

**Review by mjkkirschner — APPROVED** [review]
If all the tests pass LGTM

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**